saleor/static/dashboard-next/orders/types/OrderCancel.ts

```diff
@@ -229,6 +231,7 @@ export interface OrderCancel_orderCancel_order {
   __typename: "Order";
   id: string;
   billingAddress: OrderCancel_orderCancel_order_billingAddress | null;
+  canFinalize: boolean | null;
```

Comment:
We should change backend schema to have `canFinalize: boolean`, @dominik-zeglen do you mind if I quickly fix that on this branch?